geonode/geoapps/migrations/0006_geoapp_blob_migration.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 3.2.4 on 2021-07-05 11:21
+
+from django.contrib.contenttypes.models import ContentType
+from django.db import migrations
+import json
+
+def convert_geostory_blob(apps, _):
+    model = apps.get_model('base', 'ResourceBase')
+    for item in model.objects.filter(resource_type='geostory'):
+        if isinstance(item.blob, str):
+            new_blob = json.loads(item.blob)
+            model.objects.filter(id=item.id).update(blob=new_blob)
+        rtype = ContentType.objects.get(app_label="geoapps")
+        model.objects.filter(id=item.id).update(polymorphic_ctype=rtype)
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('geoapps', '0005_auto_20210705_1121'),
+    ]
+
+    operations = [
+        migrations.RunPython(convert_geostory_blob)
```

Comment:
Please add the revert migration to `RunPython` command (e.g.: `migrations.RunPython(convert_geostory_blob, migrations.RunPython.noop)`)